index.js

```diff
@@ -0,0 +1,104 @@
+import intitializeDocument from './modules/intitializeDocument.js';
+import displayBookListSection from './modules/displayBookListSection.js';
+import displayAddBookSection from './modules/displayAddBookSection.js';
+import displayContactSection from './modules/displayContactSection.js';
+
+class Application {
+  constructor() {
+    this.contactForm = document.querySelector('.book_form');
+    this.title = document.querySelector('.book_title');
+    this.author = document.querySelector('.book_author');
+    this.bookList = document.querySelector('.parent_book_container');
+    this.dateTime = document.querySelector('.date_time');
+    this.bookListSection = document.querySelector('.book_list_section');
+    this.addBookSection = document.querySelector('.add_book_section');
+    this.contactSection = document.querySelector('.contact_section');
+
+    this.booksArray = [];
+    this.monthMap = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  }
+
+  addBook() {
+    displayBookListSection(this.bookListSection, this.addBookSection, this.contactSection);
+    this.booksArray.push({ title: this.title.value, author: this.author.value });
+    localStorage.setItem('books', JSON.stringify(this.booksArray));
+    this.title.value = '';
+    this.author.value = '';
+    intitializeDocument(this.bookList, this.booksArray);
+/* eslint-disable */
+    intitializeRemoveButtonEvents(this);
+  }
+  // intitializeDocument()
+
+  // removeBook(index)
+
+  // intitializeRemoveButtonEvents()
+
+  // addBook()
+
+  // displayBookListSection()
+
+  // displayAddBookSection()
+
+  // displayContactSection()
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.